pkg/vault/vault.go

```diff
@@ -153,6 +159,31 @@ func (u *vault) Init() error {
 
 }
 
+// CheckReadWriteAccess will do:
+// 	- write data in the key store
+// 	- read data from the key store
+//
+// Key: mock-test-vault-unsealer
+// Data: test
+func (u *vault) CheckReadWriteAccess() error {
+	glog.Infoln("Testing the read/write access...")
+	key := "mock-test-vault-unsealer"
+	data := []byte("test")
+
+	err := u.keyStore.Set(key, data)
+	if err != nil {
+		return errors.Wrapf(err, "failed to write data for key(%s)", key)
+	}
+
+	_, err = u.keyStore.Get(key)
```

Comment:
Should you not delete here so that test data does not stay around ?

Reply:
I think you are right. Let me work on that.